app/routes.tsx

```diff
@@ -178,10 +177,11 @@ export const routes = createRoutesFromElements(
         </Route>
         <Route path="health" element={null} handle={{ crumb: 'Health' }} />
         <Route path="update" element={null} handle={{ crumb: 'Update' }} />
-        <Route path="networking" element={<NetworkingPage />}>
+        <Route path="networking">
+          <Route index element={<Navigate to="ip-pools" replace />} />
```

Comment:
Just in case, but no one should ever land on plain `/system/networking` unless they're manually messing with the URL.